src/reynir_correct/settings.py

```diff
@@ -478,9 +488,160 @@ def add(morph: str, boundlist: List[str], freelist: List[str]) -> None:
+        if wrong_form in d:
+            return Ritmyndir.DICT[wrong_form][1]
+        elif wrong_form.lower() in d:
+            return Ritmyndir.DICT[wrong_form.lower()][1]
+        else:
+            return 0
+        return Ritmyndir.DICT[wrong_form][1]
+
+    @staticmethod
+    def get_cat(wrong_form: str) -> str:
+        d = Ritmyndir.DICT
+        if wrong_form in d:
+            return Ritmyndir.DICT[wrong_form][2]
+        elif wrong_form.lower() in d:
+            return Ritmyndir.DICT[wrong_form.lower()][2]
+        else:
+            return ""
+
+    @staticmethod
+    def get_correct_form(wrong_form: str) -> str:
+        d = Ritmyndir.DICT
+        if wrong_form in d:
+            return Ritmyndir.DICT[wrong_form][3]
+        elif wrong_form.lower() in d:
+            return Ritmyndir.DICT[wrong_form.lower()][3]
+        else:
+            return ""
+
+    @staticmethod
+    def get_tag(wrong_form: str) -> str:
+        d = Ritmyndir.DICT
+        if wrong_form in d:
+            return Ritmyndir.DICT[wrong_form][4]
+        elif wrong_form.lower() in d:
+            return Ritmyndir.DICT[wrong_form.lower()][4]
+        else:
+            return ""
+
+    @staticmethod
+    def get_eink(wrong_form: str) -> int:
+        d = Ritmyndir.DICT
+        eink = 1
+        if wrong_form in d:
+            eink = Ritmyndir.DICT[wrong_form][5]
+        elif wrong_form.lower() in d:
+            eink = Ritmyndir.DICT[wrong_form.lower()][5]
+        return eink
+
+    @staticmethod
+    def get_malsnid(wrong_form: str) -> str:
+        d = Ritmyndir.DICT
+        if wrong_form in d:
+            return Ritmyndir.DICT[wrong_form][6].split(",")[0]
+        elif wrong_form.lower() in d:
+            return Ritmyndir.DICT[wrong_form.lower()][6].split(",")[0]
+        else:
+            return ""
+
+    @staticmethod
+    def get_stafs(wrong_form: str) -> str:
+        d = Ritmyndir.DICT
+        if wrong_form in d:
+            return Ritmyndir.DICT[wrong_form][7].split(",")[0]
+        elif wrong_form.lower() in d:
+            return Ritmyndir.DICT[wrong_form.lower()][7].split(",")[0]
+        else:
+            return ""
+
+    @staticmethod
+    def get_aslatt(wrong_form: str) -> str:
+        d = Ritmyndir.DICT
+        if wrong_form in d:
+            return Ritmyndir.DICT[wrong_form][8].split(",")[0]
+        elif wrong_form.lower() in d:
+            return Ritmyndir.DICT[wrong_form.lower()][8].split(",")[0]
+        else:
+            return ""
+
+    @staticmethod
+    def get_beyg(wrong_form: str) -> str:
+        d = Ritmyndir.DICT
+        if wrong_form in d:
+            return Ritmyndir.DICT[wrong_form][9].split(",")[0]
+        elif wrong_form.lower() in d:
+            return Ritmyndir.DICT[wrong_form.lower()][9].split(",")[0]
+        else:
+            return ""
+
+    @staticmethod
+    def get_code(wrong_form: str) -> str:
+        code = ""
+        w = wrong_form
+        if wrong_form not in Ritmyndir.DICT:
```

Comment:
This function can be made much more efficient by doing the Ritmyndir.DICT lookup only once, then operating on the result of that.

Reply:
Overhauled all get functions.